src/components/Pollution.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { airpollution } from '../Redux/Reducers/Pollution';
+import '../App';
+
+const Time = (element) => {
+  const date = new Date(element.props * 1000);
+  return `${date.toLocaleDateString()}\n${date.toLocaleTimeString()}`;
+};
+
+const Pollution = () => {
+  const location = useLocation();
+  const dispatch = useDispatch();
+  const info = location.state;
+  useEffect(() => {
+    dispatch(airpollution(info));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dispatch]);
```

Comment:
I noticed you disabled eslint in this file, it is against microverse policy to disable linters for linters help us write clean code. it would be nice if you enabled eslint.